Examples/Scripts/Python/geometry.py

```diff
@@ -43,6 +43,8 @@ def runGeometry(
                 raise RuntimeError("Failed to decorate event context")
 
         if outputCsv:
+            # if not os.path.isdir(outputDir + "/csv"):
+            #    os.makedirs(outputDir + "/csv")
```

Comment:
Do we need this at all?

Reply:
Yes, for the moment I uncomment it, but it's a common problem as this test requires csv/json directories to be there, but since it is unrelated, I want to do that in a second PR a little nicer.